lua/packer.lua

```diff
@@ -519,6 +550,24 @@ packer.loader_complete = function(lead, _, _)
   return completion_list
 end
 
+-- Snapshot installed plugins
+-- Given a list of plugins it will only snapshot said plugins,
+-- otherwise it will snapshot all plugins (except for disabled)
+packer.snapshot = function (filename)
```

Comment:
I might be missing something but this function describes taking a list of plugins otherwise will snap shot all plugins but where is the list it takes?

Reply:
My bad, will fix